source/guides/guides/command-line.md

```diff
@@ -287,10 +319,10 @@ cypress cache clear
 You can use also require Cypress as a Node module.
 
 {% note warning %}
-The Cypress module is brand new and we are still adding more functionality to it. Its API may change in the future.
+The Cypress module is still in beta and we are still adding more functionality to it. Its API may change in the future.
 {% endnote %}
 
-Here's an example of programmatically running a spec file:
+Here is an example of programmatically running a spec file:
```

Comment:
Well darn. I just added a contraction in another PR 😢 I'll respect this house style in the event of future PRs, sorry.